src/ui/Alert.tsx

```diff
@@ -179,7 +187,34 @@ const Alert = ({
           />
         )}
       </Inline>
-    </Stack>
+      {closable && (
+        <Button
+          variant={ButtonVariants.UNSTYLED}
+          iconName={Icons.TIMES_CIRCLE}
+          alignSelf="center"
+          css={`
+            color: ${getValue(`borderColor.${variant}`)};
+            width: 30px;
+            height: 30px;
+            display: inline-flex;
+            align-items: center;
+            justify-content: center;
+            cursor: pointer;
+            border-radius: 50%;
```

Comment:
A lot of these css attributes can be passed as props:
E.g.
`color={getValue(`borderColor.${variant}`)}`
`borderRadius="50%"`
...